ray-operator/controllers/common/constant.go

```diff
@@ -29,4 +29,7 @@ const (
 	RAY_IP         = "RAY_IP"
 	RAY_PORT       = "RAY_PORT"
 	REDIS_PASSWORD = "REDIS_PASSWORD"
+
+	// Ray core default configurations
+	DefaultRedisPassword = "5241590000000000"
```

Comment:
Just noting --
Starting with the upcoming Ray 1.11.0, Ray will by default no longer use Redis --should be possible to drop some of the logic around Redis passwords at that point.

Reply:
This is a great reminder. Let me add a note there. Once 1.11.0 is released, we can based set the configs based on ray versions.

Reply:
Add TODO to field reference in common/pod.go#50-52